classes/PodsInit.php

```diff
@@ -507,7 +507,7 @@ public function register_assets() {
 			true
 		);
 
-		wp_register_script( 'pods-cleditor', PODS_URL . 'ui/js/jquery.cleditor.min.js', array( 'jquery' ), '1.3.0' );
+		wp_register_script( 'pods-cleditor', PODS_URL . "ui/js/cleditor/jquery.cleditor{$suffix_min}.js", array( 'jquery' ), '1.4.5' );
```

Comment:
* :warning: In footer ($in_footer) is not set explicitly wp_register_script; It is recommended to load scripts in the footer. Please set this value to `true` to load it in the footer, or explicitly `false` if it should be loaded in the header.
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: b0f8e64f10042d7b1967540893f0459c</sub>